eng/common/scripts/Delete-RemoteTag.ps1

```diff
@@ -0,0 +1,33 @@
+param(
```

Comment:
@chidozieononiwu thanks for syncing the directory again. Do you know why this script addition was missed to begin with?